scripts/start_all.js

```diff
@@ -114,14 +117,149 @@ addProcess('server', async () => {
   return serverProcess;
 });
 
+addProcess('graph-node', async () => {
+  await new Promise(resolve => {
+    console.log(); // New line
+    console.log('Cleaning up the old graph-node docker data folder. For this we need', chalk.bold.red('ROOT'), 'permissions');
```

Comment:
I don't want to make this change. 

This will render the new line with a space in front of it. Try it locally and see.

It's something like this:
```
Some line

 Cleaning up the old graph...
```

Notice the empty space in front of "Cleaning up..."